views/data_summary.erb

```diff
@@ -0,0 +1,355 @@
+<div class="container" id="about">
+  <div class="page-section">
+    <div class="row">
+      <div class="column-one-quarter">
+         <%= erb :about_nav_menu %>
+      </div>
+      <div class="column-three-quarters">
+        <h1>What’s in EveryPolitician’s data?</h1>
+        <p class="section-intro">
+          Short answer: names. And dates of birth, twitter handles, politicial
+          group memberships, email addresses, honorific titles, image URLs, and
+          all sorts of useful things — all provided we can find them. How
+          complete this data is depends on the sources we’ve found for each
```

Comment:
I'm wondering if this should be turned around slightly to make it less "what *we* have found" and more "what *you* have told us about"?